shell/src/main/java/org/apache/accumulo/shell/commands/ClasspathCommand.java

```diff
@@ -30,9 +30,9 @@ public class ClasspathCommand extends Command {
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState) {
 
     final PrintWriter writer = shellState.getWriter();
-    org.apache.accumulo.start.classloader.vfs.AccumuloVFSClassLoader.printClassPath(s -> {
-      writer.print(s);
-    }, true);
+    // org.apache.accumulo.start.classloader.vfs.AccumuloVFSClassLoader.printClassPath(s -> {
+    // writer.print(s);
+    // }, true);
```

Comment:
I'm going to think about how we can still do this.

Reply:
Showing the contents of the `java.class.path` system property is probably enough here. For a configured context classloader factory, we could extend the SPI to have a method to show information about each context and call that here also. But, since this is run on the client side in the shell, it would have to be set up on the client side, and not just the server-side... unless we had a new RPC to specifically ask a random server to give us that info. And, in any case, we wouldn't be able to have much insight into the user's configured system classloader, if they set that... because that could do anything.

Reply:
I think it might be sufficient to remove the classpath command. In the default scenario, the server and client VMs will use the standard JVM classpath mechanism. In the case where a users is using the new VFS classloader, the classpath is [printed](https://github.com/apache/accumulo-classloaders/blob/main/modules/vfs-class-loader/src/main/java/org/apache/accumulo/classloader/vfs/AccumuloVFSClassLoader.java#L596) to the server-side log during initialization (so it's only printed once).

Reply:
Was the command previously deprecated? If not, we could at least dump `java.class.path` to make it *somewhat* useful without just removing it, and possibly causing scripts to break.

Reply:
I don't think so, but could be done in 2.1.1.

Reply:
So, I think that it could print `java.class.path` (of the shell, not the server, unless they are the same) and then print out all of the values of the `table.class.loader.context` properties defined on the tables.